.github/workflows/terrascan.yml

```diff
@@ -15,5 +15,5 @@ jobs:
     with:
       iac_type: terraform
       iac_dir: ./tests
-      policy_type: all
+      policy_type: azure
```

Comment:
@michaelamattes Can you please briefly explain why you changed the policy_type?

Reply:
@habr-mms we have to name the poliy_type exactly, otherwise the terrascan workflow fails because it looks for files that are not present in the repo